src/strongbox/core.clj

```diff
@@ -1232,6 +1254,23 @@
   (swap! state assoc :in-repl? (utils/in-repl?))
   nil)
 
+(defn-spec migrate-user-config nil?
+  []
+  (config/copy-wowman-config (paths :old-cfg-file) (paths :cfg-file)))
+
+(defn-spec migrate-user-catalogue nil?
+  []
+  (catalogue/copy-user-catalogue (paths :old-user-catalogue-file) (paths :user-catalogue-file)))
+
+(defn-spec migrate-user nil?
+  "migrates wowman settings and files to strongbox"
```

Comment:
add note: must be called *after* init-dirs or else new strongbox config dir may not exist as destination